rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -8561,6 +8561,16 @@ declare export class TransactionBuilder {
     output_builder: TransactionOutputAmountBuilder
   ): void;
 
+  /**
+   * @param {PlutusData} datum
+   */
+  add_extra_datum(datum: PlutusData): void;
```

Comment:
Can it be renamed into `add_extra_witness_datum` and `get_extra_witness_datums`?

Reply:
Done